src/common/atlas/cluster.ts

```diff
@@ -3,23 +3,45 @@ import type { ApiClient } from "./apiClient.js";
 import { LogId } from "../logger.js";
 
 const DEFAULT_PORT = "27017";
+
+function extractProcessIds(connectionString: string): Array<string> {
```

Comment:
```suggestion
type AtlasProcessId = "${string}:${number}"

function extractProcessIds(connectionString: string): Array<AtlasProcessId> {
```

It is more clear what the format of the string is expected, but opaque enough so can be used without knowing it.

Reply:
Will add this!